webmention.php

```diff
@@ -40,9 +40,10 @@
  * Plugin Version Number used for caching.
  */
 function version() {
-	return '5.X';
-}
+	$meta = get_plugin_meta( false );
```

Comment:
Where does this get_plugin_meta function come from? It isn't in core. And it causes the code to crash

Reply:
Fixed it myself.

Reply:
It is defined at the end of this file. it calles the core function `get_plugin_data` but it seems to be only loaded in the wp-admin context.

Reply:
I fixed it, using the function that it itself calls, which is not only in an admin context